app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] It would be nice if you make methods `index`, and `show` into your user controller because these methods are responsible for redirecting the UI and for some logical operations 🤔 So kindly add index and show methods in your users controller 👍 to follow the project requirements 😊